lua/telescope/builtin/init.lua

```diff
@@ -859,6 +859,54 @@ builtin.marks = function(opts)
   }):find()
 end
 
+builtin.registers = function(opts)
+  opts = opts or {}
+
+  local registers_table = {}
+  table.insert(registers_table, "\"")
+  table.insert(registers_table, "_")
+  table.insert(registers_table, "#")
+  table.insert(registers_table, "=")
+  table.insert(registers_table, "_")
+  table.insert(registers_table, "/")
+
+  -- selection
+  table.insert(registers_table, "*")
+  table.insert(registers_table, "+")
```

Comment:
Oh we could just do `local registers_table = { "\"", "_", "#" ...}`. That would save us a lot of inserts. I am not talking about the for loops btw. These are good.

Reply:
done